utils.go

```diff
@@ -423,19 +425,22 @@ func doesPluginSourceExist(gitURL string) (exists bool) {
 	return info.IsDir()
 }
 
-var goPath = ""
+func getGoPath() (goPath string, err error) {
+	goPath = cachedGoPath
 
-func getGoPath() (string, error) {
-	if goPath == "" {
+	if len(goPath) == 0 {
 		goEnvCmd := exec.Command("go", "env", "GOPATH")
 		output, err := goEnvCmd.CombinedOutput()
 		if err != nil {
-			return "", err
+			err = fmt.Errorf("failed to read GOPATH: %v", err)
+			return
 		}
 
-		goPath = strings.TrimSpace(string(output))
+		cachedGoPath = strings.TrimSpace(string(output))
+		goPath = cachedGoPath
```

Comment:
I'd remove line 431 and move this to before the final return statement (outside the conditional)